pulsar-common/src/main/java/org/apache/pulsar/common/protocol/Commands.java

```diff
@@ -643,6 +643,15 @@ public static ByteBuf newActiveConsumerChange(long consumerId, boolean isActive)
         return serializeWithSize(cmd);
     }
 
+    public static ByteBuf newActiveConsumerChange(long consumerId, String keySharedProps) {
```

Comment:
Can we merge this method into the above method?

Reply:
Already revised